test/e2e_node/density_test.go

```diff
@@ -189,22 +190,28 @@ var _ = framework.KubeDescribe("Density [Serial] [Slow]", func() {
 
 		for _, testArg := range dTests {
 			itArg := testArg
-			desc := fmt.Sprintf("latency/resource should be within limit when create %d pods with %v interval (QPS %d) [Benchmark]", itArg.podsNr, itArg.interval, itArg.APIQPSLimit)
-			It(desc, func() {
-				itArg.createMethod = "batch"
-				testInfo := getTestNodeInfo(f, itArg.getTestName(), desc)
+			Context("", func() {
+				desc := fmt.Sprintf("latency/resource should be within limit when create %d pods with %v interval (QPS %d) [Benchmark]", itArg.podsNr, itArg.interval, itArg.APIQPSLimit)
```

Comment:
Why not include the description directly in the `It(...)`?

Reply:
I think it's just too long, so the initial author make it a separate variable.

Reply:
It would be better to just split the string with `""+""` notation, than to make a new variable.

Reply:
We can clean this up in the future, as this pattern is repeated across the test we should be consistent with the cleanup, and I want to merge this to fix the failing density test, without blocking on nits.

Reply:
Offline discussed. There are several `desc` used in this way in this test and resource_usage_test.go.

We may want to cleanup them all together. :)